fulp_modules/batong/code/arcade_thing.dm

```diff
@@ -0,0 +1,5 @@
+/obj/machinery/computer/arcade/Initialize()
+	. = ..()
+	if(GLOB.arcade_prize_pool)
+		GLOB.arcade_prize_pool += list(/obj/item/toy/plush/batong = 1)
```

Comment:
are you 100% sure that you want this code to be run every time an arcade machine is created? what happens if the arcade machines are created before the arcade prize pool is? are you sure that this won't load the toy prize pool with tons of duplicate batong entries and/or heavily weight a single batong entry, or does += not work like that for lists?

Reply:
turns out that  GLOB.arcade_prize_pool += list overwrites the prize pool with what is between (), i can fix it with adding all the prizes to the list that overwrites, and then it works fine

Reply:
It's on the vending machine's initialize, it shouldn't have that problem

Reply:
if you make a arcade it give more % to get the baton, in one point you will get only batongs, so i change the batong to vending machines.